site/content/blog/kapp_deploy_oidc_gke.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "kapp deploy on GCP using keyless authentication(OIDC)"
+slug: kapp-deploy-oidc-gke
+date: 2022-07-14
+author: Yash Sethiya
+excerpt: "Use github action OIDC token to authenticate on GCP and deploy using kapp on GKE"
+image: /img/logo.svg
+tags: ['kapp', 'oidc', 'keyless-authentication']
+---
+
+### Who
+
+This article can be helpful for anyone who wants to create the Github Action workflow to authenticate with GCP and how to deploy Kubernetes manifest on GKE using kapp. 
+
+### Why
+
+Traditionally, authenticating from GitHub Actions to Google Cloud required exporting and storing a long-lived JSON service account key, turning an identity management problem into a secrets management problem. 
+
+But now, with GitHub's introduction of OIDC tokens into GitHub Actions Workflows, you can authenticate from GitHub Actions to Google Cloud using OIDC (Workload Identity Federation), removing the need to export a long-lived JSON service account key.
+
+### Benefits
+
+By updating workflows to use OIDC tokens, we can adopt the following good security practices:
+
+- No cloud secrets: No need to duplicate cloud credentials as long-lived GitHub secrets. Instead, you can configure the OIDC trust on GCP, and then update workflows to request a short-lived access token from the cloud provider through OIDC.
```

Comment:
Suggestion: If you want to make the points bold
- No cloud secrets: No need to duplicate cloud credentials as... =>  - **No cloud secrets:** No need to duplicate cloud credentials as...

Reply:
Done. Thanks.